packages/grant-explorer/src/features/api/utils.ts

```diff
@@ -621,6 +621,10 @@ export const pinToIPFS = (obj: IPFSObject) => {
 };
 
 export const getDaysLeft = (epochTime: number) => {
+  // Invalid date
+  if (epochTime > Number.MAX_SAFE_INTEGER) {
+    return 0;
```

Comment:
@carlbarrdahl when is this expected to happen? I feel like this should be an actual error being thrown, but not sure when it happens and if we can recover

Reply:
> @carlbarrdahl when is this expected to happen? I feel like this should be an actual error being thrown, but not sure when it happens and if we can recover

Some rounds have timestamps set to "864000000000000000000" or something like that. It's causing rendering errors. See the second card:

![](https://user-images.githubusercontent.com/87520659/275756791-ff2f1a6e-c640-4e94-bab2-3bab2e57e418.png)